src/app/containers/Home/Home.js

```diff
@@ -0,0 +1,83 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+
+import AccountInfo from '../../components/AccountInfo'
+import RenameAccount from '../../components/RenameAccount'
+import TransactionList from '../../components/TransactionList'
+
+import style from './Home.css'
+
+const Home = ({
+  neo,
+  gas,
+  label,
+  address,
+  transactionHistory,
+  onClickHandler,
+  onSubmitHandler,
+  onChangeHandler,
+  showInputField,
+  amountsError,
+  transactionHistoryError,
+  getBalance,
+  getTransactions,
+  labelError,
+  showDropDown,
+  toggleDropDownMenu,
+}) => (
+  <Fragment>
+    <section className={ style.accountInfoWrapper }>
+      <section className={ style.accountInfoContainer }>
+        {showInputField ? (
+          <RenameAccount
+            accountName={ label }
+            onSubmitHandler={ onSubmitHandler }
+            onChangeHandler={ onChangeHandler }
+            labelError={ labelError }
+          />
+        ) : (
+          <AccountInfo
+            onClickHandler={ onClickHandler }
+            neo={ neo }
+            gas={ gas }
+            label={ label }
+            address={ address }
+            amountsError={ amountsError }
+            getBalance={ getBalance }
```

Comment:
I feel like some of these could be loaded directly into the child components? Right now if feels like home is taking a lot of props.